pinot-core/src/main/java/org/apache/pinot/core/segment/index/column/BaseVirtualColumnProvider.java

```diff
@@ -39,7 +39,7 @@
 
   public DataFileReader buildReader(VirtualColumnContext context) {
     if (context.getFieldSpec().isSingleValueField()) {
-      return new ConstantSingleValueInvertedIndex(0);
+      return new ConstantSingleValueSortedIndex(0);
```

Comment:
Why is this class renamed to SortedIndex instead of InvertedIndex?

Reply:
Because it is sorted index (sorted index can be used as both forward and inverted index).
The multi-value one should be split into forward and inverted index, but that is out of the scope of this pr, so will address later.